target/linux/ipq40xx/image/Makefile

```diff
@@ -158,6 +158,22 @@ define Device/engenius_eap1300
 endef
 TARGET_DEVICES += engenius_eap1300
 
+define Device/engenius_ens620ext
+        $(call Device/FitImage)
+        DEVICE_DTS := qcom-ipq4018-ens620ext
+        DEVICE_DTS_CONFIG := config@4
+        BLOCKSIZE := 128k
+        PAGESIZE := 2048
```

Comment:
This is weird, can you please provide what flash chip your device is using? Because if it's SPI-NOR then the BLOCKSIZE and PAGESIZE are probably wrong. If it's SPI-NAND then you need ubi.

Reply:
I have not looked directly on the PCB but the uboot log shows W25Q256 (SPI NOR) :
FLASH:  SF: Detected W25Q256 with page size 64 KiB, total 32 MiB
ipq_spi: page_size: 0x100, sector_size: 0x10000, size: 0x2000000
32 MB

Looks like BLOCKSIZE should be 64k and PAGESIZE should be 256.